providers/capa/china/capa_suite_test.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func TestCAPAChina(t *testing.T) {
-	suite.Setup(false, &capa.ChinaBuilder{})
+	suite.Setup(false, "capa", &capa.ChinaBuilder{})
```

Comment:
This is needed for the lookup in Github figuring out the latest previous major release for a specific provider